layouts/partials/sidebar-single/sidebar.css

```diff
@@ -10,6 +10,10 @@
     {{ end }}
 "}" +
 
+"#{{ ($.Scratch.Get "settings").name }}.iframe-content{"+
```

Comment:
Why do we need the id selector here?

Reply:
@karloscastaneda

Reply:
@rfornes this was specifically for the jumpsport widget, *view more* button was getting chopped.

note: there was no ticket.

Reply:
Why do we need the id selector here?

Reply:
ah yeah, the actual id selector is not necessary, will remove it.

Reply:
:+1: please let me know once this has been removed thanks.

Reply:
@rfornes updated